nx/lib/nx/binary_backend.ex

```diff
@@ -2197,7 +2202,16 @@ defmodule Nx.BinaryBackend do
 
     result =
       for row <- data do
-        fft_list(row, n)
+        case kind do
+          :fft ->
+            fft_list(row, n)
+
+          :ifft ->
+            row
+            |> Enum.map(&Complex.conjugate/1)
+            |> fft_list(n)
+            |> Enum.map(&(Complex.conjugate(&1) / n))
```

Comment:
Does it mean it could be implemented using optional callbacks? Or not actually advisable?

Reply:
It could, yeah, but we would have implementations for both Torchx and EXLA, so I don't see much advantage in doing this.

I'd also expect everywhere that provides DFT to provide inverse DFT as well.

Reply:
Ship it!